core/src/main/scala/org/apache/spark/api/r/RRDD.scala

```diff
@@ -165,6 +165,11 @@ private[r] object RRDD {
   JavaRDD[Array[Byte]] = {
     PythonRDD.readRDDFromFile(jsc, fileName, parallelism)
   }
+
+  private[spark] def serveToStream(
+      threadName: String)(writeFunc: OutputStream => Unit): Array[Any] = {
+    PythonRDD.serveToStream(threadName, new RSocketAuthHelper())(writeFunc)
```

Comment:
This is required because the way of handling auth is different from RSocketAuthHelper and regular SocketAuthHelper. cc @vanzin

Reply:
Calling `PythonRDD` here is a little weird, but since that's already done in other places, looks fine.

Reply:
Yea.. I plan to pull those out into a separate class. Let me give a try in another PR maybe in the near future.

Reply:
it is. ditto `PythonRDD.readRDDFromFile`